src/components/MapEmbed.js

```diff
@@ -18,15 +21,17 @@ import { isSinglePostInEditor } from '../helpers/globals';
  * @return {JSX.Element} The rendered React component.
  */
 const MapEmbed = (props) => {
+	const isAdmin = select('core')?.canUser('create', 'posts');
+	const isPostEditor = select('core/edit-post') ? true : false;
 	const { zoom, type, className } = props;
 	let { location, height } = props;
 
 	if (!height) {
 		height = 300;
 	}
 
-	if (!isSinglePostInEditor() && !location) {
-		location = '60 29th Street #343, San Francisco, CA 94110, USA';
+	if (isAdmin && !isPostEditor && !location) {
+		location = '660 4th Street #119 San Francisco CA 94107';
```

Comment:
Need to update Jest test below with new address.

Reply:
Finally got that test updated. Since only admin should see the default address map, the test now checks for an empty container, as that is the now-expected result.